packages/suite-desktop/src-electron/electron.ts

```diff
@@ -193,6 +196,76 @@ const init = async () => {
 
     mainWindow.loadURL(src);
 
+    // TOR
+    const torSettings = store.getTorSettings();
+    const sess = mainWindow.webContents.session;
+    const toggleTor = async (start: boolean) => {
+        if (start) {
+            if (torSettings.running) {
+                await tor.start();
+            } else {
+                await tor.start();
```

Comment:
not sure why this condition is here for

Reply:
Woops, was meant to be restart :facepalm: